src/components/landing/HeadArticle.js

```diff
@@ -53,13 +55,22 @@ class HeadArticle extends Component {
         <div className="button-container text-center">
           <h2>What do you want to do?</h2>
           <div className="buttons">
-            <button
-              className="btn btn-warning"
-              onClick={this.handleClick}
-              type="button"
-            >
-              Create event
-            </button>
+            <AuthComponent onlyAnonymous>
+              <div className="text-center">
```

Comment:
Remove unnecessary div, which just wrap another element. In case if you want to specify some class with styles, you can try to assign this class directly to ModalWind